FloripaBus/SharedModule.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using Ninject.Modules;
+using System.Net.Http;
+
+namespace FloripaBus
+{
+	public class SharedModule : NinjectModule
+	{
+		public override void Load() 
+		{			
+			this.Bind<HttpClient> ().ToSelf ();
+
+			this.Bind<INavigationService> ().To<NavigationService> ();
+
+			this.Bind<IRouteRepository>().To<RouteRepository>();
+
+			this.Bind<MainViewModel> ().ToSelf ();

```

Comment:
The view model is like a part of the view with the logic which can be tested by unit tests and/or other tests. It's not expected to have another class that depends on the view model.

Your comment makes me start thinking about a view that has more than on view model and can look to a context before choosing the view model to use. Good point.